gcloud/monitoring/client.py

```diff
@@ -282,3 +283,89 @@ def list_resource_descriptors(self, filter_string=None):
             https://cloud.google.com/monitoring/api/v3/filters
         """
         return ResourceDescriptor._list(self, filter_string)
+
+    def group(self, group_id=None, name=None, display_name='', parent_name='',
```

Comment:
Shouldn't we use None values for the display name, parent name, and filter string?